src/app/rosetta/lib/account.ml

```diff
@@ -339,6 +358,27 @@ module Balance = struct
         | Some account ->
             M.return account
       in
+      let make_balance_amount ~liquid_balance ~total_balance =
+        let amount =
+          ( match token_id with
+          | None ->
+              Amount_of.coda
+          | Some token_id ->
+              Amount_of.token token_id )
+            liquid_balance
+        in
+        let locked_balance =
+          Unsigned.UInt64.sub total_balance liquid_balance
```

Comment:
@mrmr1993 Is this fine for computing the locked balance or are there any gotchas I'm not thinking about?

Reply:
This is perfect, yep!